README.md

```diff
@@ -6,6 +6,15 @@ This playbook has been tested with Ansible version 5.10.0 and above. We recommen
 
 ## Install
 
+On systems running the latest Ubuntu LTS version (Jammy), run the following commands to install Ansible:
+
+```
+sudo apt-get install ansible
```

Comment:
I believe this installs an older version of ansible- we can just refer people to the ansible installation documentation instead, or recommend to install via pip like ansible documentation recommends

Reply:
Do we have a strong reason to ask people to install the upstream version instead of the Ubuntu version? Do we use any feature that is not available?

Reply:
I think the lines bellow instructing the install of `ansible.utils` is useful, but I think it's safe to assume the user has ansible installed if they're using our playbook, and since there are so many methods to install ansible, I think it would be reasonable to specify a minimum version, but leave it to the user as to how its installed. There's also ansible-tower / AWX which makes this a bit more complicated, so best we just leave this open in my opinion.

Reply:
maybe we should move this to https://maas.io/docs/how-to-spin-up-maas-with-ansible

Reply:
I think leaving it open or directing the user to Ansible's own installation documentation should suffice

Reply:
I think it is generally not advisable to install ansible directly as a package on your system, since this usually cause a lot of issue down the line when trying to work with different ansible playbooks from different projects, which require different versions. To avoid this, i would recommend to use a virtual environment and install ansible into it. To document the needed steps, i have created the PR #122.